winui/Ribbon/Ribbon-QAT.md

```diff
@@ -131,6 +148,12 @@ To remove an items from Quick Access Toolbar (QAT), right-click the required QAT
 
 ![WinUI Ribbon QAT item removed](Ribbon-QAT-images/winui-ribbon-qat-item-removed.png)
 
+QAT item can also be removed by using Customize window, just select the QAT item in the right panel and select `Remove`. 
```

Comment:
QAT items can also be removed by using the Customize window. Select the QAT item in the right panel, then select `Remove`.

Reply:
Updated